app/views/home/small_bets.html.erb

```diff
@@ -0,0 +1,352 @@
+<header class="grid override border-b border-black grid-cols-1 bg-[#f0f0f0]">
+  <div class="relative flex items-center justify-center px-4 py-32 lg:px-[8vw] md:pb-20 md:pt-20">
+    <div class="flex flex-col max-w-screen-lg items-center gap-6 text-center z-10">
+      <%= image_tag "smallbets/smallbets.svg", alt: "Small bets", class: "h-48" %>
+      <h1 class="leading-none text-5xl md:text-7xl">You made your $1.<br>Now let's make many more.</h1>
+      <div class="text-xl max-w-md lg:text-2xl lg:max-w-3xl">Small Bets is a community of creators who support and learn from each other how to build a business, one small bet at a time.</div>
+      <div class="mt-2 flex flex-col sm:flex-row items-center justify-center gap-3 sm:gap-4 w-full max-w-[384px] sm:max-w-none sm:w-auto mx-auto">
+        <div class="w-full sm:w-auto">
+          <%= render "home/shared/button", text: "Join Small Bets", url: "https://dvassallo.gumroad.com/l/small-bets", variant: "light", size: "default" %>
+        </div>
+      </div>
+      <div class="text-lg text-muted mt-4">
+        Join <span class="font-medium text-black">6,947 amazing people</span> and get <br> immediate access to our support network.
```

Comment:
We can leave this number hardcoded for now, but at some point I'd like to provide a JSON endpoint from where we can fetch the realtime number.